### ISSUE TITLE
Sandbox and Calabash Gems

### ISSUE THREAD

**Issue by topgenorth** [calabash/install#21]
The install script for the sandbox installs the most recent gems at the point in time which the install is run. It would be nice if the calabash sandbox would report the version of the Gems installed and most recent version available:

```
$ calabash-sandbox
calabash-ios:       0.16.4 (current)
calabash-android:   0.5.15 (current)
xamarin-test-cloud: 1.1.2 (current)
This terminal is now ready to use with Calabash.
To exit, type 'exit'.
[tom@calabash-sandbox]$ 
```

It would also be convenient if the calabash-sandbox would provide a mechanism to update to the current Calabash gems, for example 

```
$ calabash-sandbox --update-gems
calabash-ios:       0.16.4 (current)
calabash-android:   0.5.15 (current)
xamarin-test-cloud: 1.1.2 (current)
This terminal is now ready to use with Calabash.
To exit, type 'exit'.
[tom@calabash-sandbox]$ 
```

A command line switch to update the Calabash gems would also be convenient:

```
$ calabash-sandbox --update-gems
Updating the Calabash gems...
######################################################################## 100.0%
Done! Installed:
calabash-ios:       0.16.999
calabash-android:   0.5.999
xamarin-test-cloud: 1.1.999
Execute 'calabash-sandbox' to get started.
```

**Comment by sapieneptus** [comment]
@jmoody While we could devise a mechanism to do this by downloading a newer bundle, perhaps we should be steering users toward creating a Gemfile etc.?

Perhaps the answer is to create a gemfile in the PWD and run `bundle install`

**Comment by jmoody** [comment]
+1 for Gemfile and bundler

I wonder if this will work as expected from inside the sandbox...it is definitely worth a try.

**Comment by sapieneptus** [comment]
We now have `calabash-sandbox update` which updates the gems. 

@jmoody How do you feel about outputting the current version info when `calabash-sandbox` is invoked?

**Comment by jmoody** [comment]
> current version

Yes!  The more information and signals that you are _in_ the sandbox the better.
